content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,730 @@
+:guilabel:`Calendar: Event Update` email template populates the email body field. The followers of
+the document (job application), as well as the :guilabel:`Recruiter` are added as
+:guilabel:`Recipients` by default. Add the applicant's email address to the list to send the email
+to the applicant as well. Make any other desired changes to the email. If an attachment is needed,
+click the :guilabel:`Attach A File` button, navigate to the file, then click :guilabel:`Open`. Once
+the email is ready to be sent, click :guilabel:`Send`.
+
+.. image:: recruitment/email-event.png
+   :align: center
+   :alt: Enter the information to send the event via email.
+
+To send the meeting via text message, click the :guilabel:`SMS` button next to the list of
+attendees. A :guilabel:`Send SMS Text Message` pop-up appears. At the top, in a blue box,
+information appears regarding the message. The first line lists how many records (phone numbers) are
+selected. The second line lists the number of recipients and how many of them are invalid. If a
+contact does not have a valid phone number listed, click :guilabel:`Close`, and edit the attendee's
+record, then redo these steps.
+
+.. warning::
+   It is *not* recommended to check the box to send to all the records in the :guilabel:`Send SMS
+   Text Message` pop-up. This would send a message to all contacts with a valid phone number.
+
+When the message reads :guilabel:`0 are invalid`, type in the message to be sent in the
+:guilabel:`Message` field. Beneath the message field, the number of characters, as well as the
+amount of text messages required to send the message (according to GSM7 criteria) appears. Click
+:guilabel:`Put In Queue` to have the text sent later, after any other messages are scheduled, or
+click :guilabel:`Send Now` to send the message immediately.
+
+.. image:: recruitment/send-sms.png
+   :align: center
+   :alt: Send a text message to the attendees of the meeting.
+
+.. note::
+   Sending text messages is not a default capability with Odoo. To send text messages, credits are
+   required, which need to be purchased. For more information on credits and plans, refer to
+   :doc:`this documentation <../marketing/sms_marketing/pricing/pricing_and_faq>`.
+
+Second interview
+----------------
+
+After an applicant has passed the :guilabel:`First Interview` stage, they next move to the
+:guilabel:`Second Interview` stage. To move the applicant to the next stage, drag and drop the
+applicant's card to the :guilabel:`Second Interview` stage, or click on the :guilabel:`Second
+Interview` stage button at the top of the individual applicant's card.
+
+When the applicant's card moves to the :guilabel:`Second Interview` stage, there are no automatic
+activities or emails configured for this stage, unlike for the :guilabel:`First Interview` stage.
+The recruiter should :ref:`schedule a second interview <recruitment/schedule>` with the applicant,
+following the same process as for the first interview.
```

Comment:
It sounds like you're telling the customer how to do their recruitment process- may instead of saying should, use can

Reply:
We offer the software to accommodate their recruitment processes-- not tell them how.

Reply:
I am telling them how the default recruitment flow is configured in Odoo and how the flow works. So if they downloaded the Recruitment app and started to recruit for a position out of the box, I wrote this for them to follow the steps created, so they understand the flow. I know they can modify it, but I think they should know how t's configured so they can make the changes being fully informed. Example- if there are automatic emails sent for a stage and they just rename it because they don't know how it works, etc- that's a problem.  I covered this in the opening paragraph, that this is how it's done out of the box but can be modified. I will try to do the 'can' language, but that's why I wrote it the way I did.